lms/djangoapps/verify_student/models.py

```diff
@@ -819,14 +820,15 @@ def download_photo_id_image(self):
 
         try:
             # decode rsa encrypted aes key from base64
-            rsa_encrypted_aes_key = base64.urlsafe_b64decode(self.photo_id_key)
+            rsa_encrypted_aes_key = base64.urlsafe_b64decode(self.photo_id_key.encode('utf-8'))
```

Comment:
Added this to ensure that decryption steps are exactly the opposite of the encryption steps